test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -79,18 +70,9 @@ public void IrisSdcaMaximumEntropy()
             var connectionString = GetDataPath(TestDatasets.iris.trainFilename);
             var commandText = "Label;SepalLength;SepalWidth;PetalLength;PetalWidth";
 
-            var loaderColumns = new DatabaseLoader.Column[]
-            {
-                new DatabaseLoader.Column() { Name = "Label", Type = DbType.Int32 },
-                new DatabaseLoader.Column() { Name = "SepalLength", Type = DbType.Single },
-                new DatabaseLoader.Column() { Name = "SepalWidth", Type = DbType.Single },
-                new DatabaseLoader.Column() { Name = "PetalLength", Type = DbType.Single },
-                new DatabaseLoader.Column() { Name = "PetalWidth", Type = DbType.Single }
-            };
+            var loader = mlContext.Data.CreateDatabaseLoader<IrisData>();
 
-            var loader = mlContext.Data.CreateDatabaseLoader(loaderColumns);
```

Comment:
Should we have at least one test covering the other CreateDatabaseLoader public methods instead of moving them all to `CreateDatabaseLoader<T>()`?

Reply:
Not sure it is worthwhile. `CreateDatabaseLoader<T>` just calls the non-generic version, so it is being tested regardless.

Reply:
Having at least 1 test for a public API is always worthwhile IMO. It ensures people don't break the public API in the future.

Reply:
Reverted `DatabaseLoaderTests.IrisLightGbm` to manually creating the columns.